homeassistant/components/starlink/switch.py

```diff
@@ -71,6 +71,7 @@ async def async_turn_off(self, **kwargs: Any) -> None:
         key="stowed",
         name="Stowed",
         device_class=SwitchDeviceClass.SWITCH,
+        entity_registry_enabled_default=False,
```

Comment:
This one seems like its not going to update frequently?

When we thinking about frequently we are generally thinking about time in seconds.

The ones we really want disabled by default are things like RSSI, voltage, etc sensors that are updating every few seconds.

Reply:
My bad, I missed switches entirely when I went over them again